src/entities.ts

```diff
@@ -25,8 +25,8 @@ export interface FeeHistoryResponse {
  */
 export interface MaxFeeSuggestions {
   maxBaseFeeSuggestion: string;
-  baseFeeTrend: number;
   currentBaseFee: string;
+  baseFeeTrend: number;
```

Comment:
@jinchung wouldn't approve it because it's not abc

Reply:
lol you're totally right